server/controllers/cohort/getCohorts.js

```diff
@@ -0,0 +1,17 @@
+const cohorts = require('../../database/models/cohort');
+
+module.exports = (req, res, next) => {
+  cohorts
+    .find({}, (err, rows) => {
```

Comment:
@alidah0 I just learned that it's possible to `await` on the query object returned by `mongoose.find()`, if you like using `await` or want to reduce the nesting in this code.  I like using `await` so it's probably a personal preference, but if you want to try, I think it would look like this:
```
try {
  const rows = cohorts.find({});
  if (!rows.length) {
    return res.status(400).json( .....
  }
  return res.status(200).json( ....
}
catch(next);
```

this post explains more:
http://thecodebarbarian.com/how-find-works-in-mongoose.html